qlib/workflow/exp.py

```diff
@@ -172,11 +184,12 @@ def end(self, recorder_status):
             self.active_recorder.end_run(recorder_status)
             self.active_recorder = None
 
-    def create_recorder(self):
-        recorders = self.list_recorders()
-        num = len(recorders)
-        name = "Recorder_{}".format(num + 1)
-        recorder = MLflowRecorder(name, self.id, self._uri)
+    def create_recorder(self, recorder_name=None):
+        if recorder_name is None:

```

Comment:
We don't have to count the number of recorders.   Duplicated recorder name is OK.
It's completely OK to keep this.